src/clients/kdump.rb

```diff
@@ -734,6 +734,8 @@ def cmdKdumpShow(options)
     end
 
 
+    # Only numbers are allowed as allow_mem_high and allow_mem_low values
+    ALLOC_MEM_REGEXP = /\D/
```

Comment:
@gilsonsouza this is not what @imobachgs has suggested. This will wrongly mark "R2D2" as valid input.